### ISSUE TITLE
Duality between 'value' attribute and property of <input> does not survive cloneNode()

### ISSUE THREAD

**Issue by kpreid** [google/caja#1793]
Original [issue 1795](https://code.google.com/p/google-caja/issues/detail?id=1795) created by ihab.awad on 2013-07-09T06:49:29.000Z:

Given 'x' an &lt;input&gt; element, it is the case that:

  x.setAttribute('value', 'xyzzy');
  x.getAttribute('value') === 'xyzzy';
  x.value === 'xyzzy';

however, if we select:

  y = x.cloneNode();

then:

  y.setAttribute('value', 'foo');
  y.getAttribute('value') === 'foo';
  y.value === 'xyzzy';

In other words, the 'value' property in the cloned node is no longer a live shadow of the 'value' attribute.
